app/dashboard/page.tsx

```diff
@@ -8,9 +9,17 @@ export default async function VideoPage() {
   // console.log(session);
 
   return (
-    <div className="flex flex-col gap-8 h-screen items-center justify-center bg-gray-100 text-black">
-      <ClientTaskList />
-      <AddTasks />
+    <div
+      className="h-screen bg-gray-100 text-black
+      grid gap-8 grid-cols-1 md:grid-cols-2"
```

Comment:
Note to self: I'm trying to break up tailwind styles into smaller groups so they're easier to read - for example here I'm defining the basic h, bg, and text settings first, and keeping all grid definitions together. 

There's probably some type of IDE extension to make this easier, a prettier config option? 🤔